hw03_gatewooda.py

```diff
@@ -0,0 +1,101 @@
+#################################################################################
+# Author: Austin Gatewood
+# Username:gatewooda
+#
+# Assignment:hw03 psychedelic turtles
+# Purpose:learn pull requests/ git in general
+# Google Doc Link:https://docs.google.com/document/d/1KOOO9_UM-Ka7Hg6IGKj9Xm-SKa6xDAZhze_jKE_fc78/edit?usp=sharing
+#
+#################################################################################
+# Acknowledgements:
+#
+#
+#################################################################################
+import turtle
+import math
+
+
+def face(radius):  # super slow creation
+    """
+    Will create the face for the 3d glasses.
+    """
+
+    happy = turtle.Turtle()  # will create the face for the glasses
+    happy.hideturtle()
+    happy.speed(0)
+    happy.color("#087209")
+
+    # Move forward to the position where the face should start
+    happy.penup()
+    happy.goto(20, -radius)
+    happy.pendown()
+
+    # Draw a circle
+    for _ in range(360):  # In the range of 360 degrees radius is doubled than multiplied by pi
```

Comment:
This is unusual. Seems like you might have pulled the idea off the web. Don't forget to cite sources.

Reply:
I had used a similar design for Hw02 and expanded on it however I did use this website provided down below, for inspiration. 
![Screenshot (1)](https://github.com/Berea-College-CSC-226/hw03-main/assets/147771488/f83d1a85-f43c-4b43-a752-2acc79d48d96)
https://www.geeksforgeeks.org/draw-circle-in-python-using-turtle/

I've added both to the acknowledgements :)